### PR TITLE
Handle Two Types of Event Counts

### PR DESCRIPTION
**Pull request by JHiltbrand**
Companion PR: https://github.com/StealthStop/Framework/pull/355

Acknowledge that there is now a *generated* event counts and *actual* event counts that could be coming from the filelist. For hadding purposes (to verify if all events in a sample were run over) we return `nActEvts` in the `sampleList` function.

### CODE REVIEW

File: Analyzer/test/condor/samples.py [hunk 1]
```diff
@@ -24,8 +25,9 @@ def nSamples(self, name):
     def sampleList(self, name):
         files = self.lib.SC_samples(self.obj, name)
         names = self.lib.SC_samples_names(self.obj, name)
-        nEvts = self.lib.SC_samples_nEvts(self.obj, name)
-        list = [(files[i],names[i],nEvts[i]) for i in xrange(self.lib.SC_samples_size(self.obj, name))]
+        nGenEvts = self.lib.SC_samples_nGenEvts(self.obj, name)
```
Comment (cmadrid1) [inline Analyzer/test/condor/samples.py:28]: I guess we don't need to use the GenEvts in general?

File: Analyzer/test/condor/samples.py [hunk 2]
```diff
@@ -24,8 +25,9 @@ def nSamples(self, name):
...
+        nActEvts = self.lib.SC_samples_nActEvts(self.obj, name)
+        list = [(files[i],names[i],nActEvts[i]) for i in xrange(self.lib.SC_samples_size(self.obj, name))]
```
Comment (cmadrid1) [inline Analyzer/test/condor/samples.py:30]: Looks good to me. Seems like no need to change anything in condorSubmit.py

**Comment by cmadrid1** [comment]
+1

File: Analyzer/test/condor/samples.py (hunk 1 above)
Comment (JHiltbrand) [reply Analyzer/test/condor/samples.py:28]: Ah, that is correct, here we do not need access to `nGenEvts` just put it for symmetry, but can remove.

File: Analyzer/test/condor/samples.py (hunk 2 above)
Comment (JHiltbrand) [reply Analyzer/test/condor/samples.py:30]: Exactly, transparent to the user.

**Pull request merged**